content/blog/elm-review-v2/index.md

```diff
@@ -0,0 +1,269 @@
+This gives you a lot of information about your direct and indirect dependencies, for instance what modules they contain and which functions and types are defined in there.
+
+With this information, rules can be much more accurate. For instance, knowing what is added to the scope when encountering `import Xyz exposing (..)` becomes possible.
+
+Added to the new information you get from the previous points, you now have sufficient knowledge to replicate the compiler's type inference logic!
+
+Example use-cases:
+
+- Report unused imports of the form `exposing (..)` or `exposing(..)` (coming soon in [`jfmengels/review-unused`](https://package.elm-lang.org/packages/jfmengels/review-unused/latest/NoUnused-Variables))
+- [Report unused dependencies](https://package.elm-lang.org/packages/jfmengels/review-unused/latest/NoUnused-Dependencies)
+- [Report dependencies with unknown or incompatible licenses](https://github.com/jfmengels/elm-review/blob/master/tests/NoInvalidLicense.elm#L21)
+
+## Visiting the README.md file
+
+The README is an integral part of the project especially for packages. For package authors and their users, it is important that everything in there is correct.
+
+This version introduces a visitor for the README, which allows you to collect data from it and to report errors for it that can be automatically fixed.
+
+Example use-cases:
+
+- [Make sure the links in your `README.md` point to the right version](https://github.com/jfmengels/review-documentation/blob/master/src/Documentation/ReadmeLinksPointToCurrentVersion.elm)
+- Make sure the links in your `README.md` point to existing modules
+- Report invalid or badly formatted Markdown content
+
+## Visiting the comments and documentation
+
+With this version, you can also look at the contents of the comments or documentation of a function or module (To be honest, this was just me forgetting to add a visitor for it, it would have been an easy addition to v1).
+
+Example use-cases (most of these are on my todo list for [`jfmengels/review-documentation`](https://package.elm-lang.org/packages/jfmengels/review-documentation/latest/)):
+
+- Making sure that the `@docs` in the module documentation are always correct and up to date
+- Report links to non-existing or non-exposed modules, or even to invalid functions/section ids
+- Reporting duplicate Markdown sections
+- Report the usage of words like `TODO`
+
+## Adding helpers
+
+Some tasks, like targeting a specific function, are harder than they should be. For instance, if you want to forbid `Foo.bar`, you'll need to handle multiple ways that the function can be called and imported, which is tedious and error-prone.
+
+I started writing a helper named [`elm-review-scope`](https://github.com/jfmengels/elm-review-scope) that deals with this problem, and makes some tasks as easy as they should be.
```

Comment:
If you still intend for `elm-review-scope` to be unstable, it might be nice to mention it here 🙂